websites/V/Viu/dist/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.0",
+	"author": {
+		"name": "zeraneL",
+		"id": "209204682156015616"
+	},
+	"url": [
+		"viu.com",
+		"www.viu.com"
+	],
+	"description": {
+		"id": "Layanan video over-the-top",
+		"en": "Over-the-top video services"

```

Comment:
fr?

Reply:
required in fr version?

Reply:
No. Fr is For real? Change these description to something better....

Reply:
Thats the official description.